modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/configuring-pci-passthrough.adoc
+
+[id="virt-adding-kernel-arguments-enable-IOMMU_{context}"]
+= Adding kernel arguments to enable the IOMMU driver
+
+To enable the IOMMU driver in the kernel, create the `MachineConfig` object file and add the kernel arguments.
+
+.Prerequisites
+* Administrative privilege to a working {product-title} cluster.
+* Intel or AMD CPU hardware.
+* Enable Intel Virtualization Technology for Directed I/O extensions or AMD IOMMU in the BIOS (Basic Input/Output System).
++
+[NOTE]
+====
+If you have an AMD CPU, specify the kernel argument as `amd_iommu=on`.
+====
```

Comment:
I found this note confusing because I didn't know what it was referring to. I think it would be clearer to include this info as part of the callout description `<3>` instead of as a note.

Reply:
Moved it.